silkrpc/core/evm_debug.cpp

```diff
@@ -293,6 +298,46 @@ boost::asio::awaitable<std::vector<DebugTrace>> DebugExecutor<WorldState, VM>::e
     co_return debug_traces;
 }
 
+template<typename WorldState, typename VM>
+boost::asio::awaitable<std::vector<DebugTraceResult>> DebugExecutor<WorldState, VM>::execute_block(const silkworm::Block& block) {
```

Comment:
This will be removed using the new approach.

Reply:
ok